README.md

```diff
@@ -26,12 +26,19 @@ And run `php composer.phar update`
 After installing, you should tell your application to load the plugin:
 
 ```php
-// in config/bootstrap.php
-Plugin::load('Cake/ElasticSearch');
+use Cake\ElasticSearch\Plugin as ElasticSearchPlugin;
 
-// If you want the plugin to automatically configure the Elastic model provider
-// and FormHelper do the following:
-Plugin::load('Cake/ElasticSearch', ['bootstrap' => true]);
+class Application extends BaseApplication
+{
+    public function bootstrap()
+    {
+        $this->addPlugin(ElasticSearchPlugin::class);
+
+        // If you want the plugin to automatically configure the Elastic model provider
+        // and FormHelper do the following:
+        // $this->addPlugin(ElasticSearchPlugin::class, [ 'bootstrap' => true ]);
```

Comment:
Shouldn't the plugin class default bootstrap hook to on?

Reply:
I just adapted the current docs from current master, in my opinion it should indeed default to true.